packages/@uppy/svelte/package.json

```diff
@@ -23,9 +23,9 @@
     "@tsconfig/svelte": "^3.0.0",
     "rollup": "^2.60.2",
     "rollup-plugin-svelte": "^7.0.0",
-    "svelte": "^3.31.0",
-    "svelte-check": "^2.0.0",
-    "svelte-preprocess": "^4.6.1"
+    "svelte": "^3.31.0|^4.0.5",
```

Comment:
These are our dev dependencies, so I think you want to switch them to always use ^4.